python/lsst/ip/diffim/subtractImages.py

```diff
@@ -69,12 +72,26 @@ class SubtractInputConnections(lsst.pipe.base.PipelineTaskConnections,
         dimensions=("instrument", "visit"),
         storageClass="ExposureCatalog",
         name="finalVisitSummary",
+        # TODO: remove on DM-39854.
+        deprecated=(
+            "Deprecated in favor of visitSummary.  Will be removed after v26."
```

Comment:
After v27?

Reply:
When I wrote this, yes, it should have been v27.  Now that RFC-945 is adopted I'm going to anticipate its implementation by just a bit and leave it as v26.